app/views/pages/get-started/build-your-first-app/building-the-user-interface.liquid

```diff
@@ -0,0 +1,195 @@
+---
+metadata:
+  title: Building the user interface
+  description: Follow these step-by-step tutorials to build a simple To do application on platformOS.
+converter: markdown
+---
+
+
+The very basic building block of any web application is it’s HTML structure. platformOS uses Liquid as a language for creating HTML templates.
+
+In this context, you can think of _Liquid_ as an addition to your HTML that will allow you to split code into smaller fragments, render some repeating parts without actually repeating the code or showing the data pulled from the database on the page.
+
+Liquid is parsed and executed **server-side**. All of the operations using Liquid are computed first on the server, and then the result of those operations is send to user browser.
+
+
+
+## Build the main application layout
+
+The main page’s HTML structure on platformOS is kept in `app/views/layouts/application.html.liquid`. Notice the file extension is `.liquid` – this is going to be a pattern for most of the files related to page’s structure. Specifically, the file named `application.html.liquid` is automatically used as a default structure for all pages in your application.
+
+Let’s start with a very simple HTML structure:
+
+#### app/views/layouts/application.html.liquid
+
+{% raw %}
+```liquid
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="utf-8">
+		<title>To do list</title>
+	</head>
+	<body>
+		<h1>My to do list</h1>
+
+		{{ content_for_layout }}
+	</body>
+</html>
+```
+{% endraw %}
+
+You may notice a special {% raw %}`{{ content_for_layout }}`{% endraw %} tag. This little thing here is Liquid.
```

Comment:
To be exact, this is liquid syntax to render a variable. In this scenario, content_for_layout is a variable that is automatically available only in the layout - not sure if we want to add more details etc, but in case we want - here it is ;) there are also other magical variables available in Pages, for example `context`

Reply:
Updated